packages/nimble/inst/tests/test-math.R

```diff
@@ -198,16 +198,16 @@ testsMatrix = list(
 
 
 set.seed(0)
-sapply(testsVaried, test_math)    ## 12
-sapply(testsBasicMath, test_math) ## 70
+ans1 <- sapply(testsVaried, test_math)    ## 12
```

Comment:
Those ans variables suppress some output if run by directly sourcing the file (or possibly via Rscript; I'd have to check).